### PR TITLE
feat: add delete all KG nodes example

### PR DESCRIPTION
**Pull request by cowan-macady**
Closing ENG-1939

### CODE REVIEW

File: examples/knowledge/cmd/delete.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+			records = append(records, records2...)
+		}
+		fmt.Println(records)
+		resp, err := clientIngest.StreamRecords(records)
+		if err != nil {
+			fmt.Println(err.Error())
+		}
+		for _, response := range resp {
+			fmt.Println(jsonp.Format(response))
+		}
```
Comment (arxeiss) [inline examples/knowledge/cmd/delete.go:49]: Why Knowledge example call Ingest?

File: examples/knowledge/cmd/root.go [hunk 2]
```diff
@@ -28,13 +28,15 @@ import (
...
 	"github.com/indykite/indykite-sdk-go/knowledge"
 )
 
 var (
-	cfgFile string
-	client  *knowledge.Client
-	jsonp   = protojson.MarshalOptions{
+	cfgFile      string
+	client       *knowledge.Client
+	clientIngest *ingest.Client
```
Comment (arxeiss) [inline examples/knowledge/cmd/root.go:38]: Same as above. Why in examples for Knowledge we have Ingest client?

File: knowledge/knowledge_read.go [hunk 3]
```diff
@@ -255,6 +260,56 @@ func parseMultipleNodesFromPaths(paths []*knowledgepb.Path) ([]*knowledgepb.Node
...
+							Type:       caser.String(node.Type),
+						},
+					},
+				},
+			}, // lint:file-ignore U1000 Ignore report
+		}
+		records = append(records, record)
+	}
+	return records, nil
+}
```
Comment (arxeiss) [inline knowledge/knowledge_read.go:311]: Can you explain what is the purpose of this method? Why it is generating some random string? This sounds kind a weird. @maaland any inputs?

**Review by arxeiss — CHANGES_REQUESTED** [review]

File: examples/knowledge/cmd/delete.go (hunk 1 above)
Comment (cowan-macady) [reply examples/knowledge/cmd/delete.go:49]: what I said yesterday:create a temporary example where both knowledge and ingest are called until the 2 packages are united with the merge of identity in  knowledge graph

File: examples/knowledge/cmd/root.go (hunk 2 above)
Comment (cowan-macady) [reply examples/knowledge/cmd/root.go:38]: for the moment, I cannot create a delete all with only knowledge or only ingest

File: knowledge/knowledge_read.go (hunk 3 above)
Comment (cowan-macady) [reply knowledge/knowledge_read.go:311]: the random is to generate an id for for record: they can be anything, their value is not used for anything else

Comment (cowan-macady) [reply knowledge/knowledge_read.go:311]: the purpose is to get all nodes of a certain type and transform them into records ready to be ingested

Comment (cowan-macady) [reply knowledge/knowledge_read.go:311]: after discussion with @maaland, I'm going to create a helper package

**Pull request closed without merging**